src/cloudant/result.py

```diff
@@ -236,10 +236,47 @@ class Result(object):
     | Iteration                     | ``limit``, ``skip`` not permitted                         |
     +-------------------------------+-----------------------------------------------------------+
 
+    List of valid Result object customization options:
+    :param bool descending: Return documents in descending key order.
+    :param endkey: Stop returning records at this specified key.
+        Not valid when used with :class:`~cloudant.result.Result` key
+        access and key slicing.
+    :param str endkey_docid: Stop returning records when the specified
+        document id is reached.
+    :param bool group: Using the reduce function, group the results to a
+        group or single row.
+    :param group_level: Only applicable if the view uses complex keys: keys
+        that are JSON arrays. Groups reduce results for the specified number
+        of array fields.
+    :param bool include_docs: Include the full content of the documents.
+    :param bool inclusive_end: Include rows with the specified endkey.
+    :param key: Return only documents that match the specified key.
+        Not valid when used with :class:`~cloudant.result.Result` key
+        access and key slicing.
+    :param list keys: Return only documents that match the specified keys.
+        Not valid when used with :class:`~cloudant.result.Result` key
+        access and key slicing.
+    :param int limit: Limit the number of returned documents to the
+        specified count.  Not valid when used with
+        :class:`~cloudant.result.Result` iteration.
+    :param int page_size: Sets the page size for result iteration.
+    :param bool reduce: True to use the reduce function, false otherwise.
+    :param int skip: Skip this number of rows from the start.
+        Not valid when used with :class:`~cloudant.result.Result` iteration.
+    :param str stale: Allow the results from a stale view to be used. This
+        makes the request return immediately, even if the view has not been
+        completely built yet. If this parameter is not given, a response is
+        returned only after the view has been built.
+    :param startkey: Return records starting with the specified key.
+        Not valid when used with :class:`~cloudant.result.Result` key
+        access and key slicing.
+    :param str startkey_docid: Return records starting with the specified
+        document ID.
+
+
     :param method_ref: A reference to the method or callable that returns
```

Comment:
I'm not sure why this is down here after a couple of empty lines.  The `method_ref` param should be at the top of the params list.  Also, it is a string so it should read `:param str method_ref`.